src/browser/api_protocol/api_handlers/interappbus.ts

```diff
@@ -0,0 +1,206 @@
+import * as apiProtocolBase from './api_protocol_base';
+import { InterApplicationBus } from '../../api/interappbus';
+import ofEvents from '../../of_events';
+import route from '../../../common/route';
+import {
+    Acker,
+    APIMessage,
+    Identity,
+    Subscriber
+} from '../../../shapes';
+
+type IABSubscriptionArgs = [string, string, string, string, number];
+
+enum SubScriptionTypes {
```

Comment:
:neckbeard: subscriptionTypes

Reply:
Changed it to `SubscriptionTypes` though